source/about-mongosync.txt

```diff
@@ -157,6 +157,31 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster might not have a consistent 
+  snapshot of the source data, and transactions might not appear atomically on 
+  the destination cluster.
+  
+  - For any continuous synchronization use cases with ``mongosync``, ensure that 
+    ``mongosync`` commits before cutting over from the source to the 
+    destination.
+
+- When used with sharded clusters, you must pause :ref:`balancers 
+  <sharding-balancing>` on both the source and the destination for the lifetime 
+  of the sync.
+  
+  - If you use ``mongosync`` for continuous synchronization, the balancers will
```

Comment:
thinking about this some more, I don't think we need this sub-bullet. I got the feedback that the "will" wording here is confusing.

we could instead have a follow-on sentence to the bullet "When used with sharded clusters" along these lines:

"Running mongosync continuously means that this condition will need to remain true indefinitely." CC @rohshar lmk your thoughts

Reply:
On second read, it might be redundant to state this again, so I'm leaning towards just taking out the sub-bullet and keeping it with just the first sentence. With just the top bullet, as is, it does emphasize that the balancers need to be stopped on both the source & destination clusters.